### PR TITLE
MDEV-16130 MDEV-16243 versioning error messages

### PR DESCRIPTION
**Pull request by midenok**
This code is submitted under the BSD-new license.

### CODE REVIEW

File: mysql-test/suite/versioning/r/alter.result [hunk 1]
```diff
@@ -76,13 +76,17 @@ t	CREATE TABLE `t` (
   `a` int(11) DEFAULT NULL
 ) ENGINE=MyISAM DEFAULT CHARSET=latin1
 alter table t add column trx_start timestamp(6) as row start;
-ERROR HY000: Table `t` is not system-versioned
```
Comment (an3l) [inline mysql-test/suite/versioning/r/alter.result]: Should in this place `--error ER_VERS_NOT_VERSIONED` be used, since we drop system versioning before?

Comment (midenok) [reply mysql-test/suite/versioning/r/alter.result]: I don't think so, because adding system versioning will not help command to succeed.

Comment (an3l) [reply mysql-test/suite/versioning/r/alter.result]: Ohh , I have read wrong error message, thought it was only about sys-versioning.

**Comment by an3l** [comment]
MDEV-16130 is closed with 647a38818a44012c2212,
Still MDEV-16243 is in review (2. commit of this PR 2db4bc5163809e9aeead240) before closing PR.

**Comment by robertbindar** [comment]
@an3l, hi, is this work merged and PR can be closed

**Comment by an3l** [comment]
@robertbindar still https://jira.mariadb.org/browse/MDEV-16243 stalled.
@midenok what do you suggest with this PR, can it be closed, since part of it is merged already?

**Pull request closed without merging**